Dockerfile_mbtci_template

```diff
@@ -72,12 +72,15 @@ RUN ARCH= && dpkgArch="$(dpkg --print-architecture)" \
       gpg --batch --keyserver hkps://keys.openpgp.org --recv-keys "$key" || \
       gpg --batch --keyserver hkps://keyserver.ubuntu.com --recv-keys "$key" ; \
     done \
-    && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/node-v$NODE_VERSION-linux-$ARCH.tar.xz" \
-    && curl -fsSLO --compressed "https://nodejs.org/dist/v$NODE_VERSION/SHASUMS256.txt.asc" \
-    && gpg --batch --decrypt --output SHASUMS256.txt SHASUMS256.txt.asc \
-    && grep " node-v$NODE_VERSION-linux-$ARCH.tar.xz\$" SHASUMS256.txt | sha256sum -c - \
-    && tar -xJf "node-v$NODE_VERSION-linux-$ARCH.tar.xz" -C /usr/local --strip-components=1 --no-same-owner \
-    && rm -rf "$GNUPGHOME" "node-v$NODE_VERSION-linux-$ARCH.tar.xz" SHASUMS256.txt.asc SHASUMS256.txt \
+    # check downloaded archive matches the checksum
+    && curl -fsSLO --compressed "https://nodejs.org/dist/v${NODE_VERSION}/node-v${NODE_VERSION}-${GOOS}-${ARCH}.tar.gz" \
+    && curl -fsSLO --compressed "https://nodejs.org/dist/v${NODE_VERSION}/SHASUMS256.txt" \
+    && grep " node-v${NODE_VERSION}-${GOOS}-${ARCH}.tar.gz\$" SHASUMS256.txt | sha256sum -c - \
+    # check downloaded checksum file signature
+    && curl -fsSLO --compressed "https://nodejs.org/dist/${NODE_VERSION}/SHASUMS256.txt.sig" \
+    && gpg --verify SHASUMS256.txt.sig SHASUMS256.txt \
+    && tar -zxv -f "node-v${NODE_VERSION}-${GOOS}-${ARCH}.tar.gz" -C /usr/local --strip-components=1 --no-same-owner \
+    && rm -rf "$GNUPGHOME" "node-v${NODE_VERSION}-${GOOS}-${ARCH}.tar.gz" SHASUMS256.txt.sig SHASUMS256.txt \
```

Comment:
All changes made on code implementing the OpenPGP checks have been done on purpose to ensure the right ordering, the maximum security and the minimum number of commands. So please do not change them unless you have a valid reason to do it. 
I push the fixed version again.